source/install.txt

```diff
@@ -36,6 +36,11 @@ Procedure
 
       .. include:: /includes/steps/install-shell-windows.rst
 
+      .. note::
```

Comment:
Would it not be better to have this blurb appear under the macOS and Linux tabs, and have the Windows-specific blurb under the Windows tab?

Reply:
Fixed.